Freetz: We build new versions (>=0.9.29) of uClibc without gnu globbing functions.
The old builds (<=0.9.28) do however contain them.

--- poptconfig.c
+++ poptconfig.c
@@ -42,7 +42,7 @@
 /*@=declundef =exportheader =incondefs =protoparammatch =redecl =type @*/
 #endif	/* __LCLINT__ */
 
-#if !defined(__GLIBC__)
+#if !defined(HAVE_GLOB_PATTERN_P)
 /* Return nonzero if PATTERN contains any metacharacters.
    Metacharacters can be quoted with backslashes if QUOTE is nonzero.  */
 static int
--- config.h.in
+++ config.h.in
@@ -37,6 +37,9 @@
 /* Define to 1 if you have the <glob.h> header file. */
 #undef HAVE_GLOB_H
 
+/* Define to 1 if you have the glob_pattern_p function. */
+#undef HAVE_GLOB_PATTERN_P
+
 /* Define if you have the iconv() function and it works. */
 #undef HAVE_ICONV
 
